doc/deployment-on-production-env.md

```diff
@@ -0,0 +1,43 @@
+# Deployment on Production Environment
+
+In this guide we will cover installation of pre-requisites on your 
+production cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes/OpenShift cluster.
```

Comment:
We can drop "OpenShift". OpenShift is Kubernetes. Plus there are many other vendors of Kubernetes

Reply:
Done